Assets/Scripts/Game/StartUp.cs

```diff
@@ -54,26 +56,33 @@ private void Start()
                 var playerData1 = playerFactory.CreatePlayerData(player1);
                 var playerData2 = playerFactory.CreatePlayerData(player2);
 
+                playerData1.Health = playerData2.Health = _maxHealth;
+
+                playerData1.Positions = new float[] { _startedPosition.position.x, _startedPosition.position.y };
+                playerData2.Positions = new float[] { _startedPosition.position.x, _startedPosition.position.y };
+
                 GameContext.CurrentGameData.PlayersData.Add(playerData1);
                 GameContext.CurrentGameData.PlayersData.Add(playerData2);
             }
 
             player1.OnScoreAdded += OnPlayer1ScoreAdded;
             player2.OnScoreAdded += OnPlayer2ScoreAdded;
 
-            var firstPlayer = CreatePlayerShip(_player1ShipPrefab, player1);
-            var secondPlayer = CreatePlayerShip(_player2ShipPrefab, player2);
+            var firstPlayer = CreatePlayerShip(_player1ShipPrefab, player1, GameContext.CurrentGameData.PlayersData[0]);
```

Comment:
use GameContext.PlayerData1 and GameContext.PlayerData2

Reply:
i don't understand